test/spec/request-strategy/JSONP-creates-script-tags.js

```diff
@@ -0,0 +1,106 @@
+var test = require('tape');
+
+var requestTimeout = 2000;
+
+test('Request strategy creates and remove script tags when using JSONP', function(t) {
+  var every = require('lodash-compat/collection/every');
+  var fauxJax = require('faux-jax');
+  var parse = require('url-parse');
+  var sinon = require('sinon');
+  var some = require('lodash-compat/collection/some');
+
+  var createFixture = require('../../utils/create-fixture');
+
+  var clock = sinon.useFakeTimers();
+  var currentURL = parse(location.href);
+  var fixture = createFixture({
+    clientOptions: {
+      dsnHost: currentURL.host
+    },
+    indexName: 'simple-JSONP-response'
+  });
+
+  var index = fixture.index;
+
+  fauxJax.install();
+
+  var initialScriptTags = document.getElementsByTagName('script');
+  var initialScriptTagsCount = initialScriptTags.length;
+
+  // check that the current state is clean
+  t.ok(
+    every(initialScriptTags, function noJSONPTag(script) {
```

Comment:
I didn't know we could use lodash functions on HtmlCollection or NodeList, but it seems so and that's pretty good news :)